src/bootstrap/src/core/builder.rs

```diff
@@ -2549,7 +2549,12 @@ impl Cargo {
         // FIXME: the guard against msvc shouldn't need to be here
         if target.is_msvc() {
             if let Some(ref cl) = builder.config.llvm_clang_cl {
-                self.command.env("CC", cl).env("CXX", cl);
+                // FIXME: There is a bug in Clang 18 when building for ARM64:
+                // https://github.com/llvm/llvm-project/pull/81849. This is
+                // fixed in LLVM 19, but can't be backported.
+                if !target.starts_with("aarch64") && !target.starts_with("arm64ec") {
```

Comment:
Where is `arm64ec` coming from? I haven't seen that target on our CI. Or is that only for local builds?

Reply:
It's currently only for local builds, but I'm planning to request promotion to tier 2 once we have everything upgraded to Server 2022 (it has a hard requirement on VS 2022).

Reply:
Ok, thanks.